### ISSUE TITLE
Microsoft Outlook support

### ISSUE THREAD

**Issue by SetyGames** [Thomas101/wmail#619]
I would love it if you add in support for [Outlook](https://outlook.live.com). It will make things easier for people out there with a mix of different email services like me. That would really complete the program.

**Comment by raven-ie** [comment]
Yeah, outlook support would be great!

**Comment by mariusapo** [comment]
yes,that would be the best,but i would say more the exchange,so we can use yahoo in it too,using just 1 inbox from just 1 site,it can be done from browser,no problem at all,but having many and have to access them all in browser...hard,having all in 1 app to check and read them all in 1 single place...now that's something

**Comment by SetyGames** [comment]
That's what I'm saying @mariusapo
